GenderPayGap.WebUI/Controllers/ManageOrganisations/ManageOrganisationsController.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Core.Models.HttpResultModels;
+using GenderPayGap.Database;
+using GenderPayGap.Database.Models;
+using GenderPayGap.WebUI.Classes;
+using GenderPayGap.WebUI.ErrorHandling;
+using GenderPayGap.WebUI.Helpers;
+using GenderPayGap.WebUI.Models.Organisation;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GenderPayGap.WebUI.Controllers.ManageOrganisations
+{
+    [Authorize(Roles = LoginRoles.GpgEmployer)]
+    [Route("manage-organisations-new")]
+    public class ManageOrganisationsController : Controller
+    {
+
+        private readonly IDataRepository dataRepository;
+
+        public ManageOrganisationsController(IDataRepository dataRepository)
+        {
+            this.dataRepository = dataRepository;
+        }
+
+        [HttpGet]
+        public IActionResult ManageOrganisationsGet()
+        {

```

Comment:
Do we want to give a 404, or just automatically redirect to the "right" version of the page? I imagine it'd be confusing to e.g. follow a link to Manage Organisations, it have swapped to the new version on live, and you get a 404 rather than the controller leading to the right place!

Reply:
For these pages, your solution works nicely.
For other pages, I think a 404 would be appropriate (e.g. if it's a new feature where the public shouldn't know that the feature exists yet).
It can sometimes get a bit messy trying to work out what the best equivalent page is (e.g. if the structure of the journey has changed) so sometimes a 404 is cleanest - but it depends on the particular case.
I think this redirect-one-way-or-the-other works nicely here.